README.md

```diff
@@ -59,58 +59,73 @@ solang compile --target solana flipper.sol
 
 Alternatively if you want to use the solang container, run:
 
+```bash
+docker run --rm -it -v $(pwd):/sources ghcr.io/hyperledger/solang compile -v -o /sources --target solana /sources/solana/flipper.sol
 ```
-docker run --rm -it -v $(pwd):/sources ghcr.io/hyperledger/solang compile -v -o /sources --target solana /sources/flipper.sol
+
+This generates a file called `flipper.json` and `flipper.so`. In order to deploy the contract code to the account
+`F1ipperKF9EfD821ZbbYjS319LXYiBmjhzkkf5a26rC`, create the private key:
```

Comment:
We are not actually creating the private key. We can either explain where these numbers come from or say `store the private key obtained from <somewhere>`. These additions will immensely help our readers.

Reply:
I changed `create` to `save`.

Reply:
I've mentioned `solana-keygen` in our docs, this is not something for the README.md